### PR TITLE
[2024.03] CB-10540 Payment batch notifications

### PR DESCRIPTION
**Pull request by PasupuletiRohini**

### CODE REVIEW

File: collections/business/general-notifications/event-general-notifications/batch-status-updated/event-general-notifications.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "id": "batch-status-updated",
+  "eventBusinessName": "Batch Status Updated",
+  "recipientType": "internal",
+  "extractor": "#this['internalUserId']",
+  "serviceAgreementExtractor": "#this['serviceAgreementId']",
+  "generalNotifications": [
+    {
+      "id": "batch-status-updated",
+      "name": "Batch Status Updated",
+      "userPreferenceEnabled": true,
```
[the remark this answers is not in the document]
Comment (PasupuletiRohini) [reply collections/business/general-notifications/event-general-notifications/batch-status-updated/event-general-notifications.json:12]: @amorimluzia , what do you mean by `OOTB, the UI will not support this notification preferences` 
We are adding a toggle for batches to enable or disable the notification preferences. We added it as part of the ticket https://backbase.atlassian.net/browse/CB-10389

[the remark this answers is not in the document]
Comment (amorimluzia) [reply collections/business/general-notifications/event-general-notifications/batch-status-updated/event-general-notifications.json:12]: hey @PasupuletiRohini thanks for sharing it. I didn't know your team worked on the FE as well. When I raised it was to share that it won't be automatically displayed on the FE, but you got it. Thanks

**Comment by PasupuletiRohini** [comment]
> @PasupuletiRohini please update the backbase-bom on the pom file to be 2024.03-LTS. Thank you

Done

File: collections/business/general-notifications/templates/batch-status-updated/template.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "batch-status-updated-in-app-notification",
+  "description": "In-app notification template for Batch Status Updated general notification",
+  "title": "Batch Status Updated",
+  "engagement": "general-notification",
+  "channel": "in-app-notification",
+  "placeholderGroups": [
+    "userProfile"
```
[the remark this answers is not in the document]
Comment (PasupuletiRohini) [reply collections/business/general-notifications/templates/batch-status-updated/template.json]: Added placeholders

**Review by amorimluzia — APPROVED** [review]

**Pull request merged**